ui/pages/confirmations/confirm/confirm.tsx

```diff
@@ -1,39 +1,47 @@
 import React, { useState } from 'react';
-
-import { Page } from '../../../components/multichain/pages/page';
 ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
 import { MMISignatureMismatchBanner } from '../../../components/app/mmi-signature-mismatch-banner';
 ///: END:ONLY_INCLUDE_IF
-
+import { Page } from '../../../components/multichain/pages/page';
+import { GasFeeContextProvider } from '../../../contexts/gasFee';
+import { TransactionModalContextProvider } from '../../../contexts/transaction-modal';
+import AdvancedGasFeePopover from '../components/advanced-gas-fee-popover';
 import { BlockaidLoadingIndicator } from '../components/confirm/blockaid-loading-indicator';
-import ScrollToBottom from '../components/confirm/scroll-to-bottom';
-import setCurrentConfirmation from '../hooks/setCurrentConfirmation';
-import syncConfirmPath from '../hooks/syncConfirmPath';
-
+import { ConfirmAlerts } from '../components/confirm/confirm-alerts';
 import { Footer } from '../components/confirm/footer';
 import { Header } from '../components/confirm/header';
 import { Info } from '../components/confirm/info';
 import { LedgerInfo } from '../components/confirm/ledger-info';
 import { Nav } from '../components/confirm/nav';
 import { PluggableSection } from '../components/confirm/pluggable-section';
-import { TransactionModalContextProvider } from '../../../contexts/transaction-modal';
-import { GasFeeContextProvider } from '../../../contexts/gasFee';
-import AdvancedGasFeePopover from '../components/advanced-gas-fee-popover';
-import EditGasFeePopover from '../components/edit-gas-fee-popover';
-import { ConfirmAlerts } from '../components/confirm/confirm-alerts';
+import ScrollToBottom from '../components/confirm/scroll-to-bottom';
 import { Title } from '../components/confirm/title';
+import EditGasFeePopover from '../components/edit-gas-fee-popover';
+import setConfirmationAlerts from '../hooks/setConfirmationAlerts';
+import setCurrentConfirmation from '../hooks/setCurrentConfirmation';
+import syncConfirmPath from '../hooks/syncConfirmPath';
+
+const Type2TxGasModal = () => {
+  return (
+    <>
+      <EditGasFeePopover />
+      <AdvancedGasFeePopover />
+    </>
+  );
+};
 
 const Confirm = () => {
   const currentConfirmation = setCurrentConfirmation();
   syncConfirmPath();
+  setConfirmationAlerts();
```

Comment:
Is the line above required to be added as part of  this PR ?

Reply:
good catch!